code/processes/metrics.q

```diff
@@ -7,6 +7,12 @@ latest:([sym:`u#`symbol$()] time:`timestamp$(); sumssize:`int$(); sumsps:`float$
 / load settings
 windows:@[value;`windows;0D00:01 0D00:05 0D01];
 enableallday:@[value;`enableallday;1b];
+tickerplanttypes:@[value;`tickerplanttypes;`tickerplant]; 
+rdbtypes:@[value;`rdbtypes;`rdb];
+tpconsleep:@[value;`tpconsleep;10]; 
+requiredprocs:@[value;`tickerplant;{value@'`rdbtypes`tickerplanttypes}]; 
```

Comment:
Think this could just be a local variable of rdbtypes,tickerplanttypes - i.e. it doesn't have to be a config variable as it is just derived from the other two